packages/@aws-cdk/core/README.md

```diff
@@ -92,6 +92,36 @@ nested stack and referenced using `Fn::GetAtt "Outputs.Xxx"` from the parent.
 
 Nested stacks also support the use of Docker image and file assets.
 
+### Breaking automatic references
+
+The automatic references created by CDK when you use resources across stacks
+are convenient, but may block your deployments if you want to remove the
+resources that are referenced in this way. You will see an error like:
+
+```
+Export Producer:ExportsOutputFnGetAtt-****** cannot be deleted as it is in use by Consumer
+```
+
+Let's say there is a Bucket in the `producer`, and the `consumer` references its
+`bucket.bucketName`. You now want to remove the bucket and run into the error above.
+
+Unblocking yourself from this is a two-step process. This is how it works:
+
+DEPLOYMENT 1: break the relationship
+
+- Make sure `consumer` no longer references `bucket.bucketName` (maybe the consumer
+  stack now uses its own bucket, or it writes to a Dynamo table, or maybe you just
+  remove the Lambda Function altogether).
+- In the `producer` class, call `this.exportAttribute(this.bucket.bucketName)`. This
```

Comment:
```suggestion
- In the `producer` class, call `stack.exportAttribute(this.bucket.bucketName)`. This
```

Reply:
People usually write in the constructor, so it's definitely `this` at that point.